cdk/lib/frontend.ts

```diff
@@ -250,6 +250,29 @@ export class Frontend extends GuStack {
       snsTopicName: "reader-revenue-dev",
     });
 
+    new GuAlarm(this, "DefaultPromotionsLoadingFailureAlarm", {
+      app,
+      alarmName: alarmName(
+        "support-frontend could not load default promo codes from S3"
+      ),
+      alarmDescription:
+        "Impact - cannot display default product promotions on the support site",
+      actionsEnabled: shouldEnableAlarms,
+      threshold: 1,
+      evaluationPeriods: 1,
+      comparisonOperator: ComparisonOperator.GREATER_THAN_OR_EQUAL_TO_THRESHOLD,
+      metric: new Metric({
+        metricName: "DefaultPromotionsLoadingFailure",
+        namespace: "support-frontend",
+        dimensionsMap: {
+          Environment: "PROD",
+        },
+        statistic: "Average",
```

Comment:
makes no difference when the threshold is 1, but in case we change the threshold makes sense to start off as Sum
```suggestion
        statistic: "Sum",
```

Reply:
👍